src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.io.*;
+import java.util.Scanner;
+import duke.Deadline;
+import duke.Task;
+import duke.Event;
+import duke.Todo;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
+
+    static final int BY_AT_INDEX = 4;
+    static final int FIND_INDEX = 5;
+    static final int TODO_INDEX = 5;
+    static final int EVENT_INDEX = 6;
+    static final int DELETE_INDEX = 7;
+    static final int DEADLINE_INDEX = 9;
+    static final int MAX_NO_OF_TASKS = 100;
+
+    final private Ui ui;
+    private TaskList tl;
+    private static Storage storage;
+    final private static Task[] tasks = new Task[MAX_NO_OF_TASKS];
+    final private static String filePath = "C:/Users/XPS/Desktop/Uni drives me crazy/Y2S2/NUS exchange/CS2113 Software Engineering/ip/duke.txt";
```

Comment:
It is good to specify a default file path. However, you could consider the situation where another user opens the application on a different machine. Would the user be allow to save to a different path than this one?